weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTable.tsx

```diff
@@ -509,6 +501,55 @@ export const CallsTable: FC<{
     history,
   ]);
 
+  const visibleColumns = useMemo(() => {
```

Comment:
i'm not sure i follow here. But if you are just using `visibleColumns` in the button, you cloud move this calculation inside the use effect